frame/society/src/lib.rs

```diff
@@ -404,6 +410,17 @@ impl<AccountId: PartialEq, Balance> BidKind<AccountId, Balance> {
 	}
 }
 
+// Note that accessing this struct isn't free. It requires a storage read,
+// but that is pretty much unavoidable.
+pub struct MaxMembersGetter<I>(sp_std::marker::PhantomData<I>);
+impl<I> Get<u32> for MaxMembersGetter<I>
+	where I: Instance,
+{
+	fn get() -> u32 {
+		MaxMembers::<I>::get()
+	}
+}
```

Comment:
Why do we need this getter struct? Does `MaxMembers::<I>::get()` not satisfy the second type arg for `BoundedVec`?

Reply:
MaxMembers doesn't implement `Get<u32>` it implements `StorageValue<...>`.
this allows to implement `Get<u32>` using MaxMembers storage.